client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -6,6 +6,7 @@ import { bindActionCreators } from 'redux';
 import { flowRight, omit, keys, pick } from 'lodash';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
+import shallowEqual from 'react-pure-render/shallowEqual';
```

Comment:
From the React doc, we should use the purecomponent version instead https://facebook.github.io/react/docs/shallow-compare.html.

I'm a bit concerned about the performance of `isEqual`, do you think I should write our own `shallowEqual`?